docs/backup-restore.md

```diff
@@ -1,134 +1,204 @@
+
+To make ScalarDB drain outstanding requests and stop accepting new requests so that a pause duration can be created, you should use [ScalarDB Server](scalardb-server.md), which uses the Scalar Admin interface, or implement the [Scalar Admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application that uses ScalarDB.
+
+By using the [Scalar Admin client tool](https://github.com/scalar-labs/scalar-admin/tree/main/java#scalar-admin-client-tool), you can pause nodes, servers, or applications that implement the Scalar Admin interface without losing ongoing transactions.
+
+{% capture notice--info %}
+**Note**
+
+When using a PITR feature, you should minimize the clock drifts between clients and servers by using clock synchronization, such as NTP. Otherwise, the time you get as the paused duration might be too different from the time in which the pause was actually conducted, which could restore the backup to a point where ongoing transactions exist.
+
+In addition, you should pause for a sufficient amount of time (for example, 10 seconds) and use the mid-time of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+{% endcapture %}
 
-#### SQLite (backup)
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-Use the [`.backup` command](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_) along with `.timeout`.
+How you create a transactionally consistent backup depends on the type of database that you're using. Select your database to see how to create a transactionally consistent backup for ScalarDB.
 
-See also [this post](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command), for example.
+<div id="tabset-2">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Cassandra2', 'tabset-2')" id="defaultOpen-2">Cassandra</button>
+  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL2', 'tabset-2')">Cosmos DB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB2', 'tabset-2')">DynamoDB</button>
+</div>
 
-#### Amazon RDS or Azure Database for MySQL/PostgreSQL (backup)
+<div id="Cassandra2" class="tabcontent" markdown="1">
 
-You can restore to any point within the backup retention period with the automated backup feature.
+Cassandra has a built-in replication feature, so you do not always have to create a transactionally consistent backup. For example, if the replication factor is set to `3` and only the data of one of the nodes in a Cassandra cluster is lost, you won't need a transactionally consistent backup because the node can be recovered by using a normal, transactionally inconsistent snapshot and the repair feature.
 
-### Backup with explicit pausing
+However, if the quorum of cluster nodes loses their data, you will need a transactionally consistent backup to restore the cluster to a certain transactionally consistent point.
```

Comment:
Can we change this sentence as follows since `snapshot` seems to be the phrase used for backups in Cassandra? We use a similar format in the `## Restore a backup` section:

```suggestion
However, if the quorum of cluster nodes loses their data, you will need a transactionally consistent backup (snapshot) to restore the cluster to a certain transactionally consistent point.
```

Reply:
Sounds good to me.